### PR TITLE
Update EIP-1046: Move back to Review and add myself as author

### PR DESCRIPTION
**Pull request by Pandapip1**

### CODE REVIEW

**Comment by Pandapip1** [comment]
CC Tommy Nicholas (@tomasienrbc), Matt Russo (@mateosu), John Zettler (@JohnZettler), Matt Condon (@shrugs), Gavin John (@Pandapip1) for review

**Review by shrugs — COMMENTED** [review]
LGTM!

**Comment by Pandapip1** [comment]
@shrugs mind approving this PR? It has to be an actual approval-type (not comment) review in order for the bot to recognize it as an approval.

File: EIPS/eip-1046.md [hunk 1]
```diff
@@ -1,87 +1,164 @@
...
-This EIP is fully backwards compatible as its implementation simply extends the functionality of ERC20 tokens and is optional.
 
-## Test Cases
-TO-DO
+This EIP is fully backwards compatible as its implementation simply extends the functionality of ERC-20 tokens and is optional.
+
+## Security Considerations
 
-## Implementation
+### SSRF
```
Comment (SamWilsn) [inline EIPS/eip-1046.md:158]: The first time an acronym is used it should be expanded.

Comment (Pandapip1) [reply EIPS/eip-1046.md]: SSRF is a well-established acronym ("Server Side Request Forgery"), and is therefore exempt from this rule.

**Review by shrugs — COMMENTED** [review]
lgtm, review-style

**Pull request merged**